news/2024/2024-03-19-changes-to-performance-points.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Changes to Performance Points
+date: 2024-03-19 18:00:00 +0000
+---
+
+Today peppy has graciously given (almost) everyone more performance points. Read on to find out why!
+
+![](/wiki/shared/news/banners/pp-sr-osu.jpg)
+
+As part of turning on performance points for osu!(lazer), there was a change in how much **bonus pp** would be awarded. In a [video by peppy and pishifat last month](https://www.youtube.com/watch?v=D7SZk8mU7Ww&t=40s), this change was mentioned to have been applied for the lazer client, and that it would be applied to stable at a later date. **Today is that day.**
+
+Apart from the [weighted sum of your top performances](/wiki/Performance_points#weightage-system), your total pp value also includes a bonus portion depending on how much you've played (in order to give new players a sense of progression). This bonus portion is 417 pp at its maximum and *used to be* achieved when you hit around 13,000 completed beatmap difficulties. That was changed to be around 1,500 difficulties.
```

Comment:
it's limited to 1000 at a higher level (due to the query being used) so might be better to say 

```suggestion
Apart from the [weighted sum of your top performances](/wiki/Performance_points#weightage-system), your total pp value also includes a bonus portion depending on how much you've played (in order to give new players a sense of progression). This bonus portion is 417 pp at its maximum and *used to be* achieved when you hit around 13,000 completed beatmap difficulties. That was changed to be around 1,000 difficulties.
```